src/utils/string.ts

```diff
@@ -22,5 +22,7 @@ export const parseLongAddressHash = (address: string) => {
 }
 
 export const hexToUtf8 = (value: string) => {
-  return value ? decodeURIComponent(value.replace(/\s+/g, '').replace(/[0-9a-f]{2}/g, '%$&')) : value
+  if (!value) return value
+  const newValue = value.startsWith('0x') ? value.substring(2) : value
+  return decodeURIComponent(newValue.replace(/\s+/g, '').replace(/[0-9a-f]{2}/g, '%$&'))
```

Comment:
decodeURIComponent might throw error, better to place in catch, if error, return origin value

Reply:
OK, I get it.

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/21/commits/4de485f045aef930b05ddbcc2c8a95ca55e9b917

Reply:
@wuyuyue 
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/21/commits/4de485f045aef930b05ddbcc2c8a95ca55e9b917